docs/docs/get-started/package-catalog.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Package catalog
+sidebar_label: Basic Concepts
+slug: '/package-catalog'
+sidebar_position: 6
+---
+
+Package catalog
+---------------
+
+The Kurtosis package catalog if the public repository of Kurtosis packages where package authors can publish their packages which later can be found and used by any user.
+
+The catalog address is [catalog.kurtosis.com][package-catalog]
+
+
+How can authors add a new package into the catalog?
+---------------------------------------------------
+
+The catalog is made up of a curated list that is stored in the `kurtosis-package-catalog.yml` file within the [kurtosis-package-catalog repository.][package-catalog-repository]
+
+If an author wants to add their package to the catalog, they should add its name on the list inside the `kurtosis-package-catalog.yml` file and send a pull request to be validated by the CI validations and approved by a Kurtosis administrator.
+
+### What validates the CI?
+
+The CI's automatic jobs validate the following for each new Pull Request created:
+
+- That there are no duplicated names within the `kurtosis-package-catalog.yml` file, there cannot be two or more packages with the same name.
+- That the package repository exists.
```

Comment:
Question: does it need to exist *publicly* too?

Reply:
yes, it has to be public

Reply:
I made the change